channelVarUtils.ml

```diff
@@ -0,0 +1,151 @@
+open Value
+open Utility
+open Pervasives (* PIPES *)
+
+(* TODO: Maybe it would be nice to have some kind of visitors for the IR?
```

Comment:
We have visitors for the IR. The visitor type and base implementation are in `ir.{ml,mli}`.